deps/build.jl

```diff
@@ -58,8 +63,7 @@ provides(AptGet,
         "libpango1.0-0" => [pango,pangocairo],
         "libglib2.0-0" => gobject,
         "libpng12-0" => libpng,
-        "libpixman-1-0" => pixman,
-        "gettext" => gettext
```

Comment:
Probably, but I suspect you need it if you want GTK+ stock messages to be translated. And not everybody speaks English out there.